### PR TITLE
feat: Setting pages UI slightly changed, in light mode

### LINKED ISSUE #434: [FEATURE] Frontend settings page UI style change request
### Description

Fontend UI is not quite impressive.
may i change it a little bit good looking than this?
if yes, can you assign this issue to me 😀

### Screenshots

image

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Shariar-Hasan**
# Fixes Issue #434 

**My PR closes #434**

# 👨‍💻 Changes proposed(What did you do ?)
- Changed layOut of the setting box (used grid)
-  changed UI colors of box background and page backgorud
- added icon in update & cancel button
- made the Ui mobile responsive

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

i tried do it with my best :)
hope you will like it.
# 📷 Screenshots

image
image

### CODE REVIEW

**Comment by Dun-sin** [comment]
@Shariar-Hasan i prefer the before, do you have a better design? else i will close it

**Comment by Shariar-Hasan** [comment]
> @Shariar-Hasan i prefer the before, do you have a better design? else i will close it

before means this one ?
image
if yes than i can make it.

else i would like to be suggested about the design, how you want it.

**Comment by Dun-sin** [comment]
you are the one who wanted a better design i was fine with the old one. which made you propose a new one that i don't which means if you don't have another better design it's best to close the PR

**Comment by Dun-sin** [comment]
@Shariar-Hasan and yes the first one you showed me in the issue is much better

**Comment by Shariar-Hasan** [comment]
Hi dear @Dun-sin 
i have changed the UI as like this now:
image

is it okay ?

File: server/.env_sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
 THIS IS JUST A FILE TO HELP YOU KNOW WHAT IS NEEDED IN THE .env FILE.
```
Comment (Dun-sin) [inline server/.env_sample:1]: Remove the change here

File: client/.eslintcache [hunk 2]
```diff
@@ -1 +1 @@
...
...eErrorCount":0,"fixableWarningCount":0,"source":null},"kdq1wl","E:\\Development\\Visual Studio\\Clone Check\\test\\Whisper\\client\\src\\pages\\Settings.jsx",["7","8","9","10","11","12"],[],{"ruleId":"13","severity":1,"message":"14","line":15,"column":1,"nodeType":"15","messageId":"16","endLine":15,"endColumn":88},{"ruleId":"13","severity":1,"message":"17","line":21,"column":1,"nodeType":"15","messageId":"16","endLine":21,"endColumn":89},{"ruleId":"13","severity":1,"message":"18","line":104,"column":1,"nodeType":"15","messageId":"16","endLine":104,"endColumn":96},{"ruleId":"13","severity":1,"message":"18","line":136,"column":1,"nodeType":"15","messageId":"16","endLine":136,"endColumn":96},{"ruleId":"13","severity":1,"message":"19","line":153,"column":1,"nodeType":"15","messageId":"16","endLine":153,"endColumn":84},{"ruleId":"13","severity":1,"message":"18","line":165,"column":1,"nodeType":"15","messageId":"16","endLine":165,"endColumn":96},"max-len","This line has a length of 87. Maximum allowed is 80.","Program","max","This line has a length of 88. Maximum allowed is 80.","This line has a length of 95. Maximum allowed is 80.","This line has a length of 83. Maximum allowed is 80."]
```
Comment (Dun-sin) [inline client/.eslintcache:1]: Remove the change here

File: client/src/pages/Settings.jsx [hunk 3]
```diff
@@ -1,13 +1,5 @@
 import { useMemo } from 'react';
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:1]: Remove all format changes here

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by Dun-sin** [comment]
> Hi dear @Dun-sin 
> i have changed the UI as like this now:
> image
> 
> is it okay ?

Stop saying dear

File: client/.eslintcache (hunk 2 above)
Comment (Shariar-Hasan) [reply client/.eslintcache]: well i didnt write it, maybe it is auto generated  ☹️

File: client/.eslintcache
Comment (Dun-sin) [inline client/.eslintcache:1]: this file shouldn't be here at all

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by Dun-sin** [comment]
@Shariar-Hasan you also need to resolve the merge conflict

File: client/.eslintcache
Comment (Shariar-Hasan) [reply client/.eslintcache:1]: so, should i delete this and commit ?
sorry for asking so many question, as i  am newbie in git related issue ☹️

Comment (Dun-sin) [reply client/.eslintcache:1]: you can google `how to remove a change in a PR`

Comment (Dun-sin) [reply client/.eslintcache:1]: should be a good learning experience

Comment (Shariar-Hasan) [reply client/.eslintcache:1]: okay 🥺🥺

Comment (Shariar-Hasan) [reply client/.eslintcache:1]: i am messed up,
dont know what i am doing :)
You can unassign me. 🙂
i think i cant do it, i tried many way everytime i commits, .eslintcache file auto generated by eslint :)

Comment (Shariar-Hasan) [reply client/.eslintcache:1]: i tried to copy and paste the changes made, 
@Dun-sin

Comment (Dun-sin) [reply client/.eslintcache:1]: Deleting a file is also a change

Comment (Shariar-Hasan) [reply client/.eslintcache:1]: i hope, now its fixed 🙂🙂

**Comment by Dun-sin** [comment]
> @Shariar-Hasan you also need to resolve the merge conflict

@Shariar-Hasan

**Comment by Shariar-Hasan** [comment]
> > @Shariar-Hasan you also need to resolve the merge conflict
> 
> @Shariar-Hasan

hey @Dun-sin 
is this fixed now ?
😀

File: client/src/pages/Settings.jsx [hunk 4]
```diff
@@ -61,145 +61,152 @@ const Searching = () => {
...
+							<Form.HelpText className="text-zinc-600 text-xs">
+								{darkMode ? 'Dark' : 'Light'} mode
+							</Form.HelpText>
+						</div>
+						<div className="flex items-center">
+							<Form.Control
+								className="flex justify-end items-center"
+								name="theme"
+								accepter={Toggle}
+								checked={settings.theme}
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:90]: @Shariar-Hasan when you resolve merge conflicts, you just don't remove the one that you don't want especially if there is a contraction between two codes, that's the lazy way, the good way is to try and understand what the other code does and see if it needs to be kept or removed.

```suggestion
								checked={settings.theme ?? true}
```

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by Dun-sin** [comment]
@Shariar-Hasan when you resolve merge conflicts, you just don't remove the one that you don't want especially if there is a contraction between two codes, that's the lazy way, the good way is to try and understand what the other code does and see if it needs to be kept or removed.

File: client/src/pages/Settings.jsx [hunk 5]
```diff
@@ -61,145 +61,152 @@ const Searching = () => {
...
+									icon="ion:invert-mode"
+									height="18"
+									width="18"
+									className="mr-1.5"
+								/>
+								<Form.ControlLabel className="text-primary  dark:text-light  text-[12px] sm:text-xs md:text-base font-normal">
+									MODE
+								</Form.ControlLabel>
+							</div>
+							<Form.HelpText className="text-zinc-600 text-xs">
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:81]:
```suggestion
							<Form.HelpText className="text-zinc-600 text-xs text-white">
```

Comment (Dun-sin) [inline client/src/pages/Settings.jsx:81]:
```suggestion
							<Form.HelpText className="text-zinc-600 text-xs text-white">
```

Comment (Dun-sin) [inline client/src/pages/Settings.jsx:81]:
```suggestion
							<Form.HelpText className="text-zinc-600 text-xs text-white">
```

File: client/src/pages/Settings.jsx [hunk 6]
```diff
@@ -61,145 +61,152 @@ const Searching = () => {
...
+									icon="bxs:bell-ring"
+									height="18"
+									width="18"
+									className="mr-1.5"
+								/>
+								<Form.ControlLabel className="text-primary  dark:text-light text-[12px] sm:text-xs md:text-base font-normal">
+									NOTIFICATION
+								</Form.ControlLabel>
+							</div>
+							<Form.HelpText className="text-zinc-600 text-xs">
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:130]:
```suggestion
							<Form.HelpText className="text-zinc-600 text-xs text-white">
```

File: client/src/pages/Settings.jsx [hunk 7]
```diff
@@ -61,145 +61,152 @@ const Searching = () => {
...
+								icon="ic:baseline-volume-up"
+								height="20"
+								width="19"
+								className="mr-1"
+							/>
+							<Form.ControlLabel className="text-primary  dark:text-light  text-[12px] sm:text-xs md:text-base font-normal">
+								VOLUME LEVEL
+							</Form.ControlLabel>
+						</div>
+						<Form.HelpText className="text-zinc-600 text-xs mb-4">
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:159]:
```suggestion
						<Form.HelpText className="text-zinc-600 text-xs mb-4 text-white">
```

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Review by Dun-sin — APPROVED** [review]
thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Comment by Shariar-Hasan** [comment]
> @Shariar-Hasan when you resolve merge conflicts, you just don't remove the one that you don't want especially if there is a contraction between two codes, that's the lazy way, the good way is to try and understand what the other code does and see if it needs to be kept or removed.

got it 😀



> thanks for contributing, I hope you will stick around and continue to contribute to this project.
> 
> Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

thanks for giving me time and tips to solve the issue,
i have starred the projects,
and already joined in discord .
❤️

**Pull request merged**